src/misc/mmio.c

```diff
@@ -176,23 +176,26 @@ static complex float* load_cfl_internal(const char* name, unsigned int D, long d
 		return load_zcoo(name, D, dimensions);
 
 
+	char err_str[1024];
+	sprintf(err_str, "Loading cfl file %s", name);
```

Comment:
You are right. A stupid simple option is to increase the size of `err_str`.

If we are to use format strings, how would we pass it to `perror` without first allocating memory for a temporary string anyway? It seems it would just move the memory allocation to a different function?